js/cmb2.js

```diff
@@ -220,12 +221,41 @@ window.CMB2 = window.CMB2 || {};
 			// Loop through each attachment
 			$( attachment ).each( function() {
 				if ( this.type && this.type === 'image' ) {
-					var width = media.previewSize[0] ? media.previewSize[0] : 50;
-					var height = media.previewSize[1] ? media.previewSize[1] : 50;
+					
+					// Preview size dimensions
+					var previewWidth  = media.previewSize[0] ? media.previewSize[0] : 150;
+					var previewHeight = media.previewSize[1] ? media.previewSize[1] : 150;
```

Comment:
Looks like you're changing this default as well (from 50 to 15), which again, is problematic for back-compatibility.